README.md

```diff
@@ -145,8 +145,41 @@ JSON::Matchers.schema_root = "docs/api/schemas"
 
 ## Contributing
 
-1. Fork it ( https://github.com/[my-github-username]/json-matchers/fork )
-2. Create your feature branch (`git checkout -b my-new-feature`)
-3. Commit your changes (`git commit -am 'Add some feature'`)
-4. Push to the branch (`git push origin my-new-feature`)
-5. Create a new Pull Request
+Please see [CONTRIBUTING].
+
+`json-matchers` was written is maintained by Sean Doyle.
+
+Many improvements and bugfixes were contributed by the [open source community].
+
+[CONTRIBUTING]: https://github.com/thoughtbot/json-matchers/blob/master/CONTRIBUTING.md
+[open source community]: https://github.com/thoughtbot/json-matchers/graphs/contributors
+
+## Need Help?
+
+We offer 1-on-1 coaching. We can help you set up a new Rails application, write
```

Comment:
I pulled this directly from [suspenders](https://github.com/thoughtbot/suspenders#need-help).

Should we remove this line from all library READMEs?